test/support/test_helpers.ex

```diff
@@ -22,4 +22,31 @@ defmodule CodeCorps.TestHelpers do
     assert String.to_integer(result["id"]) == id
     result
   end
+
+  def put_id(payload, id), do: put_in(payload, ["data", "id"], id)
+  def put_attributes(payload, attributes), do: put_in(payload, ["data", "attributes"], attributes)
+
+  def put_relationships(payload, record_1, record_2), do: put_relationships(payload, [record_1, record_2])
```

Comment:
Super minor nitpick, but since this has different arity than the function below, should this be grouped up above with the other one-liners?